requirements/requirements.trocr.txt

```diff
@@ -0,0 +1,3 @@
+transformers
+accelerate
```

Comment:
we should probably just make a transformers requirement if we have torch already and start merging these into the main dockerfile. Just a note for the future.